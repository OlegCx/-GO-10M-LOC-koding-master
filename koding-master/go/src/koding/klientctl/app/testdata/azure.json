{
  "provider": {
    "azure": {
      "publish_settings": "${var.azure_publish_settings}",
      "subscription_id": "${var.azure_subscription_id}"
    }
  },
  "resource": {
    "azure_instance": {
      "azure-instance": {
        "custom_data": "\necho \"hello world!\" >> /helloworld.txt\n",
        "image": "Ubuntu Server 14.04 LTS",
        "name": "koding-${var.koding_group_slug}-${var.koding_stack_id}-${count.index+1}",
        "size": "Basic_A1"
      }
    }
  }
}
