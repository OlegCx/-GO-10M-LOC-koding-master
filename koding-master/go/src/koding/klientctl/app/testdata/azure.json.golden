{
  "provider": {
    "azure": {
      "publish_settings": "${var.azure_publish_settings}",
      "subscription_id": "${var.azure_subscription_id}"
    }
  },
  "resource": {
    "azure_instance": {
      "azure-instance": {
        "custom_data": "#cloud-config\nruncmd:\n- echo \"hello world!\" >> /helloworld.txt\nssh_authorized_keys:\n- ssh-rsa AAAAB3NzaC1yc2EA... koding-350298856\n",
        "image": "Ubuntu Server 14.04 LTS",
        "koding_always_on": true,
        "name": "koding-${var.koding_group_slug}-${var.koding_stack_id}-${count.index+1}",
        "size": "Basic_A1"
      }
    }
  }
}
